src/useApolloNetworkStatus.ts

```diff
@@ -130,33 +130,36 @@ function reducer(
 
 const initialState: NetworkStatus = {
   numPendingQueries: 0,
-  numPendingMutations: 0
+  numPendingMutations: 0,
+  queryError: undefined,
+  mutationError: undefined
```

Comment:
Adding `undefined` properties here as well is useful for discovery, as the user can `console.log` the network status and immediately sees what properties are available.